dataline-api/src/main/openapi/config.yaml

```diff
@@ -996,6 +1010,8 @@ components:
         - workspaceId
         - destinationSpecificationId
         - connectionConfiguration
+        - name
+        - destionationName
```

Comment:
typo

Reply:
why are there two names here?

Reply:
destinationName is the name of the destination Postgres (or maybe Singer-Postgres). name is the name of the source impl e.g. "rldb source"